lib/kubernetes-deploy/kubernetes_resource/cloudsql.rb

```diff
@@ -2,8 +2,8 @@ module KubernetesDeploy
   class Cloudsql < KubernetesResource
     TIMEOUT = 5.minutes
 
-    def initialize(name, namespace, file)
-      @name, @namespace, @file = name, namespace, file
+    def initialize(name, namespace, context, file)
+      @name, @namespace, @context, @file = name, namespace, context, file
```

Comment:
It seems like this could be the default `initialize` behaviour in `KubernetesResource` that this class inherits from.